### PR TITLE
Update department_store.json

### PR DESCRIPTION
**Pull request by arnowrld**

### CODE REVIEW

File: data/brands/shop/department_store.json [hunk 1]
```diff
@@ -218,6 +218,18 @@
...
+    {
+      "displayName": "Burlington Coat Factory",
+      "id": "burlingtoncoatfactory-0a21d9",
+      "locationSet": {"include": ["pr", "us"]},
+      "tags": {
+        "brand": "Burlington Coat Factory",
+        "brand:wikidata": "Q4999220",
+        "brand:wikipedia": "en:Burlington (department store)",
+        "name": "Burlington Coat Factory",
+        "shop": "clothes"
```
Comment (arch0345) [inline data/brands/shop/department_store.json:230]: If you want to tag this as `shop=clothes` this should be moved to the existing `shop/clothes` category
```suggestion
        "shop": "department_store"
```

File: data/brands/shop/department_store.json [hunk 2]
```diff
@@ -1318,6 +1318,18 @@
...
+    {
+      "displayName": "Winners",
+      "id": "winners-2047be",
+      "locationSet": {"include": ["ca", "es"]},
+      "tags": {
+        "brand": "Winners",
+        "brand:wikidata": "Q845257",
+        "brand:wikipedia": "en:Winners",
+        "name": "Winners",
+        "shop": "clothes"
```
Comment (arch0345) [inline data/brands/shop/department_store.json:1330]:
```suggestion
        "shop": "department_store"
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

File: data/brands/shop/department_store.json (hunk 1 above)
Comment (tas50) [reply data/brands/shop/department_store.json:230]: Another note on Burlington. They're just using the name "Burlington" now. We should set "Burlington Coat Factory" as an alt_name.

**Comment by tas50** [comment]
I updated all the Lord & Taylor stores in the US to disused except the ones in NYC. Some NYC folks have strong opinions in OSM and I want to let them survey and update those locations on their own.

**Pull request merged**